website/views/shared/tasks/task.jade

```diff
@@ -1,4 +1,4 @@
-li(bindonce='list', id='task-{{::task.id}}', ng-repeat='task in obj[list.type+"s"] | conditionalOrderBy: list.view=="dated":"date"', class='task {{Shared.taskClasses(task, user.filters, user.preferences.dayStart, user.lastCron, list.showCompleted, main)}}', ng-click='spell && (list.type != "reward") && castEnd(task, "task", $event)', ng-class='{"cast-target":spell && (list.type != "reward"), "locked-task":obj._locked === true}', popover-trigger='mouseenter', data-popover-html="{{task.notes | markdown}}", popover-placement="top", popover-append-to-body='{{::modal ? "false":"true"}}', ng-show='shouldShow(task, list, user.preferences)')
+li(bindonce='list', id='task-{{::task.id}}', ng-repeat='task in obj[list.type+"s"] | conditionalOrderBy: list.view=="dated":"date"', class='task {{Shared.taskClasses(task, user.filters, user.preferences.dayStart, user.lastCron, list.showCompleted, main)}}', ng-click='spell && (list.type != "reward") && castEnd(task, "task", $event)', ng-class='{"cast-target":spell && (list.type != "reward"), "locked-task":obj._locked === true}', popover-trigger='click', data-popover-html="{{task.notes | markdown}}", popover-placement="top", popover-append-to-body='{{::modal ? "false":"true"}}', ng-show='shouldShow(task, list, user.preferences)')
```

Comment:
### Normal Behavior: 
![screen shot 2015-05-01 at 10 12 48 pm](https://cloud.githubusercontent.com/assets/2916945/7439769/9314529e-f04f-11e4-9df1-fed96d8e9900.png)

### Click on notes icon
![screen shot 2015-05-01 at 10 12 34 pm](https://cloud.githubusercontent.com/assets/2916945/7439768/93133058-f04f-11e4-83bb-081b58e92004.png)

Now the mouseover version of the popover disappears and this persistent one takes its place, hovering over the notes icon. Clicking the note icon again makes the persistent popover vanish and the normal mouseover behavior returns.

@lemoness How does that sound to you?